build.cmd

```diff
@@ -189,429 +148,256 @@ set "__TestRootDir=%__RootBinDir%\tests"
-    echo %__MsgPrefix%System.Private.CoreLib successfully built.
-    if not defined __CoreLibAlsoNativeImage (
-        exit /b 0
+if %_buildNativeCoreLib% EQU 1 (
+    echo %__MsgPrefix%Generating native image of System.Private.CoreLib for %__BuildOS%.%__BuildArch%.%__BuildType%
+	
+    echo "%__CrossgenExe%" /Platform_Assemblies_Paths "%__BinDir%" /out "%__BinDir%\System.Private.CoreLib.ni.dll" "%__BinDir%\System.Private.CoreLib.dll"
+    "%__CrossgenExe%" /Platform_Assemblies_Paths "%__BinDir%" /out "%__BinDir%\System.Private.CoreLib.ni.dll" "%__BinDir%\System.Private.CoreLib.dll" > "%__CrossGenCoreLibLog%" 2>&1
+    if NOT %errorlevel% == 0 (
+        echo %__MsgPrefix%Error: CrossGen System.Private.CoreLib build failed. Refer to the build log file for details:
+        echo     %__CrossGenCoreLibLog%
+        exit /b 1
     )
-)
 
-echo %__MsgPrefix%Generating native image of System.Private.CoreLib for %__BuildOS%.%__BuildArch%.%__BuildType%
+    echo %__MsgPrefix%Generating native image of MScorlib facade for %__BuildOS%.%__BuildArch%.%__BuildType%
 
-set "__CrossGenCoreLibLog=%__LogsDir%\CrossgenCoreLib_%__BuildOS%__%__BuildArch%__%__BuildType%.log"
-set "__CrossgenExe=%__CrossComponentBinDir%\crossgen.exe"
-"%__CrossgenExe%" /Platform_Assemblies_Paths "%__BinDir%" /out "%__BinDir%\System.Private.CoreLib.ni.dll" "%__BinDir%\System.Private.CoreLib.dll" > "%__CrossGenCoreLibLog%" 2>&1
-if NOT %errorlevel% == 0 (
-    echo %__MsgPrefix%Error: CrossGen System.Private.CoreLib build failed. Refer to the build log file for details:
-    echo     %__CrossGenCoreLibLog%
-    type     %__CrossGenCoreLibLog%
-    exit /b 1
+    set "__CrossGenCoreLibLog=%__LogsDir%\CrossgenMSCoreLib_%__BuildOS%__%__BuildArch%__%__BuildType%.log"
+    set "__CrossgenExe=%__CrossComponentBinDir%\crossgen.exe"
+    "%__CrossgenExe%" /Platform_Assemblies_Paths "%__BinDir%" /out "%__BinDir%\mscorlib.ni.dll" "%__BinDir%\mscorlib.dll" > "%__CrossGenCoreLibLog%" 2>&1
+    if NOT %errorlevel% == 0 (
+        echo %__MsgPrefix%Error: CrossGen mscorlib facade build failed. Refer to the build log file for details:
+        echo     %__CrossGenCoreLibLog%
+        exit /b 1
+    )
 )
 
-echo %__MsgPrefix%Generating native image of MScorlib facade for %__BuildOS%.%__BuildArch%.%__BuildType%
+if %_buildPackages% EQU 1 (
 
-set "__CrossGenCoreLibLog=%__LogsDir%\CrossgenMSCoreLib_%__BuildOS%__%__BuildArch%__%__BuildType%.log"
-set "__CrossgenExe=%__CrossComponentBinDir%\crossgen.exe"
-"%__CrossgenExe%" /Platform_Assemblies_Paths "%__BinDir%" /out "%__BinDir%\mscorlib.ni.dll" "%__BinDir%\mscorlib.dll" > "%__CrossGenCoreLibLog%" 2>&1
-if NOT %errorlevel% == 0 (
-    echo %__MsgPrefix%Error: CrossGen mscorlib facade build failed. Refer to the build log file for details:
-    echo     %__CrossGenCoreLibLog%
-    type     %__CrossGenCoreLibLog%
-    exit /b 1
-)
+    set __msbuildLog=/flp:Verbosity=normal;LogFile="%__LogsDir%\Nuget_%__BuildOS%__%__BuildArch%__%__BuildType%.log"
```

Comment:
Could you please use the same naming convention for all variables? Before your change, the variable names started with two underscores and then an uppercase letter. Now some of them start with single underscore and some of them with lower case letter after the underscore.